xml/ha_config_basics.xml

```diff
@@ -36,61 +36,270 @@
+       </para>
+      </formalpara>
+     </listitem>
+    </varlistentry>
+    <varlistentry><!-- 3.3 -->
+     <term>Odd number of nodes in three locations (alternative 1)</term>
+     <listitem>
+      <formalpara>
+       <title>Configuration:</title>
+       <para>2&times;N+1 nodes, FC SAN and layer 2 network across all locations.
+       </para>
+      </formalpara>
+      <formalpara>
+       <title>Usage scenario:</title>
+       <para>
+        Variant of <xref linkend="vl.n-nodes-3locs"/>. Rarely used.
+       </para>
+      </formalpara>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec.ha.config.basics.global">
+  <title>Quorum Determination</title>
+  <para>
+   Whenever communication fails between one or more nodes and the rest of the
+   cluster, a cluster partition occurs. The nodes can only communicate with
+   other nodes in the same partition and are unaware of the separated nodes.
+   A cluster partition is defined to have quorum (is <quote>quorate</quote>)
+   if it has the majority of nodes (or votes).
+   How this is achieved is done by <emphasis>quorum calculation</emphasis>.
+   Quorum is a requirement for fencing.
+   </para>
+   <para>
+   Quorum calculation has changed between &productname;&nbsp;11 and
+   &productname;&nbsp;15. For &productname;&nbsp;11, quorum was calculated by
+   &pace;.
+   Starting with &productname;&nbsp;12, &corosync; can handle quorum for
+   two-node clusters directly without changing the &pace; configuration.
+  </para>
+
+  <para>How quorum is calculated is influenced by the following factors:</para>
+   <variablelist>
+    <varlistentry xml:id="vl.ha.config.basics.global.number-of-cluster-nodes">
+     <term>Number of Cluster Nodes</term>
+     <listitem>
+         <para>To keep services running, a cluster with more than two nodes
+       relies on quorum (majority vote) to resolve cluster partitions.
+       Based on the following formula, you can calculate the minimum
+       number of operational nodes required for the cluster to function:</para>
+      <figure>
+       <title>Formula to Calculate the Number of Operational Nodes</title>
+       <mediaobject>
+        <textobject>
+         <screen>N ≥ C/2 + 1
+
+N = minimum number of operational nodes
+C = number of cluster nodes</screen>
+        </textobject>
+       </mediaobject>
+      </figure>
+      <para>For example, a five-node cluster needs a minimum of three operational
+       nodes (or two nodes which can fail). </para>
+      <para>
+       We strongly recommend to use either a two-node cluster or an odd number
+       of cluster nodes.
+       Two-node clusters make sense for stretched setups across two sites.
+       Clusters with an odd number of nodes can be built on either one single
+       site or might be spread across three sites.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Global Cluster Options</term>
```

Comment:
These relate to the behaviors under the circumstances of with/without quorum,  but they don't affect "Quorum Determination" itself :)  How about to put it after this section to still be a separate section like before?

Reply:
@gao-yan Sorry for the late response, was busy with other issues.

Hmn, yes, probably that item is not directly related to "Quorum Determination". I'll move it into another section.

Reply:
Thanks. To clarify, I meant the introduction of this term "Global Cluster Options" and also the both paragraphs "Global Option stonith-enabled" and "Global Option no-quorum-policy".